src/components/Facet/FacetContainer.js

```diff
@@ -36,11 +36,11 @@ class FacetContainer extends React.Component {
       fetchDataPrediction,
     } = this.props;
 
-    if (facets[key].data === false && key === "Data Points") {
```

Comment:
Maybe we should use a variable which defines the name instead of setting it everywhere manually ;)

Reply:
this is also applies for the other "names"